_posts/2019-07-1-foreman-community-newsletter---june-2019.md

```diff
@@ -0,0 +1,155 @@
+and as usual the Foreman/Pulp booth looked incredibly busy every time the expo
+area open. The talks by Tomer and Ewoud were well received. As Tomer
+mentioned in the community demo, a lot of questions were asked around Ansible
+integration, working of foreman with Ansible Tower, Remote execution and also
+acknowledged the improvements to content view publishing, better UX for repos
+and audit. Sounds like good news to us! All the work that we have been putting
+in is being noticed :)
+
+Not to forget, we had the very awesome yellow Foreman hats distributed at the Foreman
+booth. Thanks to Tomer and Ori for handling the design and shipment of the goodies.
+
+### Community Survey analysis is now out
+
+Thanks to everyone who filled out this year's community survey - we have now
+the analysis of the report that was prepared by Greg. We had 116 replies, which is a
+little down on last year, but still more than enough data to work with. As promised, Greg has
+written us a blog, which you can read [here][gregs_blog].
+
+[gregs_blog]: https://theforeman.org/2015/02/foreman-community-survey-results.html
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
+* [Foreman Birthday Party](https://community.theforeman.org/t/foreman-birthday-party/13731)
+
+It's Foreman's 10th birthday soon! On 15th July, 2009, [Ohad Levy](https://github.com/ohadlevy) created an empty Rails project and then named it as the Foreman project. Since then Foreman gives system administrators the power to easily automate tasks and manage the lifecycle of thier servers effectively. To know all the great stories about this project and where it is leading, attend the Foreman Birthday Party on July 25th, 2019. Save the date for the  Foreman’s 10th Birthday Party organized by NETWAYS, ATIX, and the Foreman Project!
+
+#### [Past Events](https://community.theforeman.org/c/events/l/latest)
+
+* [OpenExpo Europe](https://community.theforeman.org/t/openexpo-europe/13467)
+
+Tomer gave a talk about “Hybrid cloud management with Foreman” at the conference.
+He spoke about some of the main features of Foreman, and how you can leverage it to simplify and unify your datacenter management.
+
+* [Foreman Meetup: Time for your first Open Source Contribution](https://community.theforeman.org/t/foreman-meetup-time-for-your-first-open-source-contribution/13012)
+
+This meetup was focused towards new-commers to join the community. It was mostly around the
+Foreman documentation followed by a discussion around how we can make it better as a community. We did have @lzap, at the meetup as he was in Pune at the same time, to speak about the importance and history of open source! s
```

Comment:
there is an "s" at the end of the sentence, not sure if it's leftover or a sentence that got cut off somehow